app.js

```diff
@@ -342,6 +345,10 @@ define(function(require) {
 						$dropdown
 							.toggleClass('show');
 
+						container
+							.find('.review-table-wrapper')
+								.attr('style', 'min-height: ' + minHeight);
```

Comment:
```suggestion
								.css('minHeight', minHeight);
```

Reply:
There are two other instance of this in your changes, please update them as well to use `jquery#css` (you can unset properties by passing an empty string).